runtime/shiden/src/lib.rs

```diff
@@ -777,6 +799,145 @@ impl pallet_ethereum::Config for Runtime {
     type StateRoot = pallet_ethereum::IntermediateStateRoot<Self>;
 }
 
+parameter_types! {
+    pub CouncilMotionDuration: BlockNumber = 36 * HOURS;
+}
+
+type CouncilCollective = pallet_collective::Instance1;
+impl pallet_collective::Config<CouncilCollective> for Runtime {
+    type Origin = Origin;
+    type Event = Event;
+    type Proposal = Call;
+    type MotionDuration = CouncilMotionDuration;
+    type MaxProposals = ConstU32<100>;
+    type MaxMembers = ConstU32<10>;
+    type DefaultVote = pallet_collective::PrimeDefaultVote;
+    type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    pub const TechnicalCommitteeMotionDuration: BlockNumber = 36 * HOURS;
+}
+
+type TechnicalCommitteeCollective = pallet_collective::Instance2;
+impl pallet_collective::Config<TechnicalCommitteeCollective> for Runtime {
+    type Origin = Origin;
+    type Event = Event;
+    type Proposal = Call;
+    type MotionDuration = TechnicalCommitteeMotionDuration;
+    type MaxProposals = ConstU32<100>;
+    type MaxMembers = ConstU32<10>;
+    type DefaultVote = pallet_collective::PrimeDefaultVote;
+    type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
+}
+
+parameter_types! {
+    pub const ProposalBond: Permill = Permill::from_percent(5);
+    pub const ProposalBondMinimum: Balance = 100 * SDN;
+    pub const SpendPeriod: BlockNumber = 1 * DAYS;
```

Comment:
Should be 3 days according to the initial plan

Reply:
Changed to 3 days